scalameta/trees/shared/src/main/scala/scala/meta/internal/Scaladoc.scala

```diff
@@ -48,6 +48,45 @@ object Scaladoc {
   /** A heading */
   final case class Heading(level: Int, title: String) extends Term
 
+  /**
+   * A table
+   * [[https://www.scala-lang.org/blog/2018/10/04/scaladoc-tables.html]]
+   */
+  final case class Table(
+      header: Table.Row,
+      align: Seq[Table.Align],
+      row: Table.Row*
```

Comment:
Would it make sense to use Seq here instead of varargs? My personal rule of thumb is to avoid varargs where possible because they make it harder to introduce new fields in the future.

Reply:
will done. chose earlier for brevity. based on this comment, submitted #2062.